packages/sdk-core/package.json

```diff
@@ -0,0 +1,32 @@
+{
+    "name": "@backtrace/sdk-core",
+    "version": "0.0.1",
+    "description": "Backtrace-JavaScript SDK core library",
+    "main": "lib/index.js",
+    "types": "lib/index.d.ts",
+    "scripts": {
+        "test": "NODE_ENV=test jest"
```

Comment:
we are missing build, rebuild etc. scripts here

Reply:
done